credentials/generate-revocation-set.py

```diff
@@ -508,7 +617,7 @@ def main(use_main_net_dcld: str, use_test_net_dcld: str, use_main_net_http: bool
         entry = {
             "type": "revocation_set",
             "issuer_subject_key_id": certificate_akid_hex,
-            "issuer_name": certificate_authority_name_b64,
+            "issuer_name": certificate_authority_name.rfc4514_string(),
```

Comment:
Note: i've updated the output to provide the human readable issuer_name.

Reply:
Looks good.